### PR TITLE
Feat/grid nft

### PR DESCRIPTION
**Pull request by iagocarvalhodev**
creation of the marketplace nft card

### CODE REVIEW

File: src/components/marketplace/CardNft.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
+import { LoadingOutlined } from '@ant-design/icons'
+import { Skeleton, Spin } from 'antd'
+import 'antd/dist/antd.css'
+import React from 'react'
+import styled from 'styled-components'
+import notFund from '../../assets/notFund.png'
```
Comment (fabriciomirandabr) [inline src/components/marketplace/CardNft.tsx:6]: wrong english

File: src/components/marketplace/CardNft.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import { LoadingOutlined } from '@ant-design/icons'
+import { Skeleton, Spin } from 'antd'
+import 'antd/dist/antd.css'
```
Comment (fabriciomirandabr) [inline src/components/marketplace/CardNft.tsx:3]: You can't import this in component

File: src/components/marketplace/CardNft.tsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+import React from 'react'
+import styled from 'styled-components'
+import notFund from '../../assets/notFund.png'
+
+export interface CardNftProps {
+  imgUrl?: string
+
```
Comment (fabriciomirandabr) [inline src/components/marketplace/CardNft.tsx:10]: Why these extra spaces?

File: src/components/marketplace/CardNft.tsx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  name?: string
+
+  price: number
+
+  loading: boolean
+
+  onClick?: () => void
+}
+export const CardNft: React.FC<CardNftProps> = ({ imgUrl = '', name = 'NFT Name', price = 3.25, loading = true }: CardNftProps) => {
```
Comment (fabriciomirandabr) [inline src/components/marketplace/CardNft.tsx:19]: Why you are initialing the values here?

File: src/components/marketplace/CardNft.tsx [hunk 5]
```diff
@@ -0,0 +1,114 @@
+import { LoadingOutlined } from '@ant-design/icons'
+import { Skeleton, Spin } from 'antd'
+import 'antd/dist/antd.css'
+import React from 'react'
+import styled from 'styled-components'
+import notFund from '../../assets/notFund.png'
+
+export interface CardNftProps {
```
Comment (fabriciomirandabr) [inline src/components/marketplace/CardNft.tsx:8]: The name should be

ITEM + TYPE 

NftCardProps

File: src/components/marketplace/CardNft.tsx [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+  imgUrl?: string
+
+  name?: string
```
Comment (fabriciomirandabr) [inline src/components/marketplace/CardNft.tsx:11]: The name should not be optional

File: src/components/marketplace/CardNft.tsx [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+
+  onClick?: () => void
+}
+export const CardNft: React.FC<CardNftProps> = ({ imgUrl = '', name = 'NFT Name', price = 3.25, loading = true }: CardNftProps) => {
+  const antIcon = <LoadingOutlined style={{ fontSize: 56 }} spin />
+  return (
+    <S.Card className='storybook-card'>
+      <S.Image>
+        <img src={imgUrl || notFund} alt={name} hidden={loading} />
+        <Spin indicator={antIcon} spinning={loading} />
```
Comment (fabriciomirandabr) [inline src/components/marketplace/CardNft.tsx:25]: spinner

File: src/components/marketplace/CardNft.tsx [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+  name?: string
+
+  price: number
+
+  loading: boolean
+
+  onClick?: () => void
+}
+export const CardNft: React.FC<CardNftProps> = ({ imgUrl = '', name = 'NFT Name', price = 3.25, loading = true }: CardNftProps) => {
+  const antIcon = <LoadingOutlined style={{ fontSize: 56 }} spin />
```
Comment (fabriciomirandabr) [inline src/components/marketplace/CardNft.tsx:20]: Put the style on the style and import from there

 on footer:   S.Icon

File: src/components/marketplace/CardNft.tsx [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+  return (
+    <S.Card className='storybook-card'>
+      <S.Image>
+        <img src={imgUrl || notFund} alt={name} hidden={loading} />
+        <Spin indicator={antIcon} spinning={loading} />
+      </S.Image>
+      <S.Info>
+        <Skeleton loading={loading} paragraph={{ rows: 1 }}>
+          <span className='storybook-title'>{name}</span>
```
Comment (fabriciomirandabr) [inline src/components/marketplace/CardNft.tsx:29]: don't need class here, use from styled components

File: src/components/marketplace/CardNft.tsx [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+        </Skeleton>
+        <div>
+          <Skeleton loading={loading} paragraph={{ rows: 1 }}>
```
Comment (fabriciomirandabr) [inline src/components/marketplace/CardNft.tsx:32]: styled(Skeleton)` styles...

File: src/components/marketplace/CardNft.tsx [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+    flex-direction: column;
+    justify-content: flex-start;
+    .storybook-skeleton-price {
+      .ant-skeleton-content {
+        display: flex;
+        justify-content: flex-end;
+      }
+    }
+    .ant-skeleton {
+      height: 20px !important;
```
Comment (fabriciomirandabr) [inline src/components/marketplace/CardNft.tsx:90]: avoid import, copy all classpath to replace

File: src/components/marketplace/CardNft.tsx [hunk 12]
```diff
@@ -0,0 +1,114 @@
...
+      .ant-skeleton-title {
+        margin-top: 0px !important;
```
Comment (fabriciomirandabr) [inline src/components/marketplace/CardNft.tsx:92]: avoid import, copy all classpath to replace

File: src/components/marketplace/CardNft.tsx (hunk 4 above)
Comment (fabriciomirandabr) [inline src/components/marketplace/CardNft.tsx:19]: NftCard

File: src/components/marketplace/CardNft.tsx (hunk 1 above)
Comment (iagocarvalhodev) [reply src/components/marketplace/CardNft.tsx:6]: done

File: src/components/marketplace/CardNft.tsx (hunk 2 above)
Comment (iagocarvalhodev) [reply src/components/marketplace/CardNft.tsx:3]: done

File: src/components/marketplace/CardNft.tsx (hunk 3 above)
Comment (iagocarvalhodev) [reply src/components/marketplace/CardNft.tsx:10]: done

File: src/components/marketplace/CardNft.tsx (hunk 4 above)
Comment (iagocarvalhodev) [reply src/components/marketplace/CardNft.tsx:19]: done

File: src/components/marketplace/CardNft.tsx (hunk 5 above)
Comment (iagocarvalhodev) [reply src/components/marketplace/CardNft.tsx:8]: done

File: src/components/marketplace/CardNft.tsx (hunk 6 above)
Comment (iagocarvalhodev) [reply src/components/marketplace/CardNft.tsx:11]: done

File: src/components/marketplace/CardNft.tsx (hunk 4 above)
Comment (iagocarvalhodev) [reply src/components/marketplace/CardNft.tsx:19]: done

**Pull request closed without merging**